package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "engelsystem",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/engelsystem/engelsystem.git",
+  "author": "marudor <marudor@marudor.de>",
+  "license": "MIT",
+  "scripts": {
+    "build": "NODE_ENV=production webpack",
+    "build:watch": "webpack --watch"
+  },
+  "dependencies": {
+    "bootstrap": "^3.3.7",
+    "bootstrap-datepicker": "^1.7.1",
+    "chart.js": "^1.0.2",
+    "icomoon": "^1.0.0",
```

Comment:
Currently the icomoon font is a stripped down version that only contains the angel logo, that would load the whole font?
For reference: #426 (Feature: Logo and text should be configurable)